### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by timhallinflux**
security vulnerability text.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,3 +15,9 @@ cd docs.influxdata.com/
 * Sign the [CLA](https://www.influxdata.com/legal/cla/)
 * Submit a pull request
 * Earn the respect, admiration, and eternal love of the entire InfluxData community
+
+### Security Vulnerability Reporting
+InfluxData takes security and our users' trust very seriously. If you believe you have found a security issue in any of our
+open source projects, please responsibly disclose by contacting us at security@influxdata.com. More details about
```
Comment (sanderson) [inline CONTRIBUTING.md]: ...responsibly disclose **it** by...

Comment (timhallinflux) [reply CONTRIBUTING.md]: fixed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,6 +18,6 @@ cd docs.influxdata.com/
 
 ### Security Vulnerability Reporting
 InfluxData takes security and our users' trust very seriously. If you believe you have found a security issue in any of our
-open source projects, please responsibly disclose by contacting us at security@influxdata.com. More details about 
+open source projects, please responsibly disclose it by contacting us at security@influxdata.com. More details about
```
Comment (kelseiv) [inline CONTRIBUTING.md]: please contact security@....

Comment (timhallinflux) [reply CONTRIBUTING.md]: responsibly disclose is the "usual" verbiage.  I removed the "us at".

**Pull request merged**